### ISSUE TITLE
🔗 🐞 Frontend template rendering is broken with fatal error when trying to get the Installation ID (Duplicate of #29579)

### ISSUE THREAD

**Issue by Flow-CH** [elementor/elementor#29582]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Hello, every morning I check on my website and I get a fatal error text below. The problem is solved as soon as I log on WordPress. If you know why and how to fix it, I would really appreciate your help. Best regards, Laurence.

```txt

Fatal error: Uncaught Error: Call to a member function get_component() on null in ../wp-content/plugins/elementor/includes/api.php:129 
Stack trace: 
#0 ../wp-content/plugins/elementor/includes/api.php(85): Elementor\Api::get_site_key() 
#1 ../wp-content/plugins/elementor/includes/api.php(178): Elementor\Api::get_info_data() 
#2 ../wp-content/plugins/elementor/modules/promotions/module.php(53): Elementor\Api::get_promotion_widgets() 
#3 ../wp-includes/class-wp-hook.php(324): Elementor\Modules\Promotions\Module->Elementor\Modules\Promotions\{closure}(Object(Elementor\Widgets_Manager)) 
#4 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array) 
#5 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array) 
#6 ../wp-content/plugins/elementor/includes/managers/widgets.php(146): do_action('elementor/widge...', Object(Elementor\Widgets_Manager)) 
#7 ../wp-content/plugins/elementor/includes/managers/widgets.php(345): Elementor\Widgets_Manager->init_widgets() 
#8 ../wp-content/plugins/elementor/includes/elements/column.php(1038): Elementor\Widgets_Manager->get_widget_types('heading') 
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(1502): Elementor\Element_Column->_get_default_child_type(Array) 
#10 ../wp-content/plugins/elementor/includes/base/element-base.php(369): Elementor\Element_Base->get_child_type(Array) 
#11 ../wp-content/plugins/elementor/includes/base/element-base.php(1547): Elementor\Element_Base->add_child(Array) 
#12 ../wp-content/plugins/elementor/includes/base/element-base.php(314): Elementor\Element_Base->init_children() 
#13 ../wp-content/plugins/elementor/core/files/css/post.php(225): Elementor\Element_Base->get_children() 
#14 ../wp-content/plugins/elementor/core/files/css/post.php(318): Elementor\Core\Files\CSS\Post->add_controls_stack_style_rules(Object(Elementor\Element_Column), Array, Array, Array, Array) 
#15 ../wp-content/plugins/elementor/core/files/css/post.php(297): Elementor\Core\Files\CSS\Post->render_element_styles(Object(Elementor\Element_Column)) 
#16 ../wp-content/plugins/elementor/core/files/css/post.php(226): Elementor\Core\Files\CSS\Post->render_styles(Object(Elementor\Element_Column)) 
#17 ../wp-content/plugins/elementor/core/files/css/post.php(318): Elementor\Core\Files\CSS\Post->add_controls_stack_style_rules(Object(Elementor\Element_Section), Array, Array, Array, Array) 
#18 ../wp-content/plugins/elementor/core/files/css/post.php(297): Elementor\Core\Files\CSS\Post->render_element_styles(Object(Elementor\Element_Section)) 
#19 ../wp-content/plugins/elementor/core/files/css/post.php(178): Elementor\Core\Files\CSS\Post->render_styles(Object(Elementor\Element_Section)) 
#20 ../wp-content/plugins/elementor/core/files/css/base.php(687): Elementor\Core\Files\CSS\Post->render_css() 
#21 ../wp-content/plugins/elementor/core/files/base.php(180): Elementor\Core\Files\CSS\Base->parse_content() 
#22 ../wp-content/plugins/elementor/core/files/css/base.php(133): Elementor\Core\Files\Base->update_file() 
#23 ../wp-content/plugins/elementor/core/files/css/base.php(229): Elementor\Core\Files\CSS\Base->update() 
#24 ../wp-content/plugins/elementor/core/files/css/post.php(201): Elementor\Core\Files\CSS\Base->enqueue() 
#25 ../wp-content/plugins/elementor/includes/frontend.php(695): Elementor\Core\Files\CSS\Post->enqueue() 
#26 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->enqueue_styles('') 
#27 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array) 
#28 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array) 
#29 ../wp-includes/script-loader.php(2272): do_action('wp_enqueue_scri...') 
#30 ../wp-includes/class-wp-hook.php(324): wp_enqueue_scripts('') 
#31 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array) 
#32 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array) 
#33 ../wp-includes/general-template.php(3064): do_action('wp_head') 
#34 ../wp-content/themes/astra/header.php(31): wp_head() 
#35 ../wp-includes/template.php(810): require_once('...') 
#36 ../wp-includes/template.php(745): load_template('...', true, Array) 
#37 ../wp-includes/general-template.php(48): locate_template(Array, true, true, Array) 
#38 ../wp-content/themes/astra/page.php(20): get_header() 
#39 ../wp-includes/template-loader.php(106): include('...') 
#40 ../wp-blog-header.php(19): require_once('...') 
#41 ../index.php(17): require('...') 
#42 {main} thrown in ../wp-content/plugins/elementor/includes/api.php on line 129
```

### Steps to reproduce

It happens overnight, every night. I've just solve it today as I log on Wordpress. 

### Expected behavior

I would like to have my website running smoothly without having to log on WP to solve the fatal error message on my website.  

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.4.24
	PHP Version: 8.0.30
	PHP Memory Limit: 1280M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 300M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.7.1
	Site URL: ..
	Home URL: ..
	WP Multisite: No
	Max Upload Size: 300 Mo
	Memory limit: 1280M
	Max Memory limit: 1280M
	Permalink Structure: /%postname%/
	Language: fr_FR
	Timezone: Europe/Zurich
	Debug Mode: Inactive

== Theme ==
	Name: Astra
	Version: 4.8.7
	Author: Brainstorm Force
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: fr-FR
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.3.1 Safari/605.1.15

== Active Plugins ==
	Astra Pro
		Version: 4.8.7
		Author: Brainstorm Force

	Elementor
		Version: 3.26.0-dev5
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.3
		Author: Elementor.com

	Premium Starter Templates
		Version: 4.4.9
		Author: Brainstorm Force

	Spectra
		Version: 2.18.0
		Author: Brainstorm Force

	Ultimate Addons for Elementor Pro
		Version: 1.37.3
		Author: Brainstorm Force

	WPForms Lite
		Version: 1.9.2.3
		Author: WPForms

	WP Mail SMTP
		Version: 4.3.0
		Author: WP Mail SMTP

	WP Portfolio
		Version: 1.12.0
		Author: Brainstorm Force

	WP Super Cache
		Version: 1.12.4
		Author: Automattic

	WPvivid Backup Plugin
		Version: 0.9.110
		Author: WPvivid Backup & Migration

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast

	Yoast SEO
		Version: 24.0
		Author: Team Yoast


== Expériences d’Elementor ==
	Polices d’icônes « inline »: Inactif
	Points d’arrêt personnalisés supplémentaires: Actif
	Conteneur: Actif
	Mettre à niveau la bibliothèque Swiper: Actif par défaut
	Performances des éléments imbriqués: Actif par défaut
	Balisage optimisé: Inactif par défaut
	Intégration d’extension: Actif par défaut
	CSS Smooth Scroll: Actif par défaut
	Écran d’accueil Elementor: Actif par défaut
	Page d’atterrissage: Inactif
	Éléments imbriqués: Inactif
	Barre supérieure de l’éditeur: Inactif
	Mise en cache d’éléments: Actif par défaut
	Lien en bio: Actif par défaut
	Boutons flottants: Actif par défaut
	Évènements de l’éditeur Elementor: Inactif par défaut
	Atomic Widgets: Inactif par défaut
	Global Classes: Inactif par défaut
	Launchpad Checklist: Inactif par défaut


== Journal ==
	
Log: showing 20 of 342024-05-23 12:00:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.0-dev3',
  'to' => '3.22.0-dev3',
)]
2024-05-23 12:00:09 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.0-dev3',
  'to' => '3.22.0-dev3',
)]
2024-11-17 08:25:49 [info] elementor::elementor_updater Started 
2024-11-17 08:25:49 [info] Elementor/Upgrades - _on_each_version Start  
2024-11-17 08:25:49 [info] Elementor/Upgrades - _on_each_version Finished 
2024-11-17 08:25:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.0-dev3',
  'to' => '3.25.0-dev3',
)]
2024-12-02 16:24:23 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.0-dev3',
  'to' => '3.26.0-dev3',
)]
2024-12-02 16:24:23 [info] elementor::elementor_updater Started 
2024-12-02 16:24:23 [info] Elementor/Upgrades - _on_each_version Start  
2024-12-02 16:24:23 [info] Elementor/Upgrades - _on_each_version Finished 
2024-12-02 16:24:23 [info] Elementor/Upgrades - _v_3_26_0 Start  
2024-12-02 16:24:23 [info] Elementor/Upgrades - _v_3_26_0 Finished 
2024-12-02 16:24:23 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.0-dev3',
  'to' => '3.26.0-dev3',
)]
2024-12-02 16:24:24 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.0-dev3',
  'to' => '3.26.0-dev3',
)]
2024-12-14 09:32:05 [info] elementor::elementor_updater Started 
2024-12-14 09:32:05 [info] Elementor/Upgrades - _on_each_version Start  
2024-12-14 09:32:05 [info] Elementor/Upgrades - _on_each_version Finished 
2024-12-14 09:32:05 [info] Elementor/Upgrades - _v_3_26_0 Start  
2024-12-14 09:32:05 [info] Elementor/Upgrades - _v_3_26_0 Finished 
2024-12-14 09:32:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.26.0-dev3',
  'to' => '3.26.0-dev5',
)]

JS: showing 3 of 3JS: 2024-03-01 10:09:44 [error X 322][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.20.0-dev3:3:403861] undefined is not an object (evaluating &#039;B[0].controls&#039;) 
JS: 2024-12-05 08:08:49 [error X 1874][../wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.8.2:2:17922] null is not an object (evaluating &#039;this.getColorRepresentation().toLowerCase&#039;) 
JS: 2024-12-05 08:36:20 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.26.0-dev3:3:634336] undefined is not an object (evaluating &#039;this.view.container.isDesignable&#039;) 

PHP: showing 5 of 5PHP: 2024-12-02 16:39:56 [warning X 1][../wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-03 04:37:36 [error X 43][../wp-content/plugins/elementor/includes/api.php::129] Uncaught Error: Call to a member function get_component() on null in ../wp-content/plugins/elementor/includes/api.php:129
Stack trace:
#0 ../wp-content/plugins/elementor/includes/api.php(85): Elementor\Api::get_site_key()
#1 ../wp-content/plugins/elementor/includes/api.php(178): Elementor\Api::get_info_data()
#2 ../wp-content/plugins/elementor/modules/promotions/module.php(53): Elementor\Api::get_promotion_widgets()
#3 ../wp-includes/class-wp-hook.php(324): Elementor\Modules\Promotions\Module-&gt;Elementor\Modules\Promotions\{closure}(Object(Elementor\Widgets_Manager))
#4 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(NULL, Array)
#5 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#6 ../wp-content/plugins/elementor/includes/managers/widgets.php(146): do_action(&#039;elementor/widge...&#039;, Object(Elementor\Widgets_Manager))
#7 ../wp-content/plugins/elementor/includes/managers/widgets.php(345): Elementor\Widgets_Manager-&gt;init_widgets()
#8 ../wp-content/plugins/elementor/includes/elements/column.php(1038): Elementor\Widgets_Manager-&gt;get_widget_types(&#039;heading&#039;)
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(1502): Elementor\Element_Column-&gt;_get_default_child_type(Array)
#10 ../wp-content/plugins/elementor/includes/base/element-base.php(369): Elementor\Element_Base-&gt;get_child_type(Array)
#11 ../wp-content/plugins/elementor/includes/base/element-base.php(1547): Elementor\Element_Base-&gt;add_child(Array)
#12 ../wp-content/plugins/elementor/includes/base/element-base.php(314): Elementor\Element_Base-&gt;init_children()
#13 ../wp-content/plugins/elementor/core/files/css/post.php(225): Elementor\Element_Base-&gt;get_children()
#14 ../wp-content/plugins/elementor/core/files/css/post.php(318): Elementor\Core\Files\CSS\Post-&gt;add_controls_stack_style_rules(Object(Elementor\Element_Column), Array, Array, Array, Array)
#15 ../wp-content/plugins/elementor/core/files/css/post.php(297): Elementor\Core\Files\CSS\Post-&gt;render_element_styles(Object(Elementor\Element_Column))
#16 ../wp-content/plugins/elementor/core/files/css/post.php(226): Elementor\Core\Files\CSS\Post-&gt;render_styles(Object(Elementor\Element_Column))
#17 ../wp-content/plugins/elementor/core/files/css/post.php(318): Elementor\Core\Files\CSS\Post-&gt;add_controls_stack_style_rules(Object(Elementor\Element_Section), Array, Array, Array, Array)
#18 ../wp-content/plugins/elementor/core/files/css/post.php(297): Elementor\Core\Files\CSS\Post-&gt;render_element_styles(Object(Elementor\Element_Section))
#19 ../wp-content/plugins/elementor/core/files/css/post.php(178): Elementor\Core\Files\CSS\Post-&gt;render_styles(Object(Elementor\Element_Section))
#20 ../wp-content/plugins/elementor/core/files/css/base.php(687): Elementor\Core\Files\CSS\Post-&gt;render_css()
#21 ../wp-content/plugins/elementor/core/files/base.php(180): Elementor\Core\Files\CSS\Base-&gt;parse_content()
#22 ../wp-content/plugins/elementor/core/files/css/base.php(133): Elementor\Core\Files\Base-&gt;update_file()
#23 ../wp-content/plugins/elementor/core/files/css/base.php(229): Elementor\Core\Files\CSS\Base-&gt;update()
#24 ../wp-content/plugins/elementor/core/files/css/post.php(201): Elementor\Core\Files\CSS\Base-&gt;enqueue()
#25 ../wp-content/plugins/elementor/includes/frontend.php(695): Elementor\Core\Files\CSS\Post-&gt;enqueue()
#26 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend-&gt;enqueue_styles(&#039;&#039;)
#27 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(NULL, Array)
#28 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#29 ../wp-includes/script-loader.php(2272): do_action(&#039;wp_enqueue_scri...&#039;)
#30 ../wp-includes/class-wp-hook.php(324): wp_enqueue_scripts(&#039;&#039;)
#31 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(&#039;&#039;, Array)
#32 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#33 ../wp-includes/general-template.php(3064): do_action(&#039;wp_head&#039;)
#34 ../wp-content/themes/astra/header.php(31): wp_head()
#35 ../wp-includes/template.php(810): require_once(&#039;...&#039;)
#36 ../wp-includes/template.php(745): load_template(&#039;...&#039;, true, Array)
#37 ../wp-includes/general-template.php(48): locate_template(Array, true, true, Array)
#38 ../wp-content/themes/astra/page.php(20): get_header()
#39 ../wp-includes/template-loader.php(106): include(&#039;...&#039;)
#40 ../wp-blog-header.php(19): require_once(&#039;...&#039;)
#41 ../index.php(17): require(&#039;...&#039;)
#42 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-04 05:43:16 [error X 1][../wp-content/plugins/elementor/includes/api.php::129] Uncaught Error: Call to a member function get_component() on null in ../wp-content/plugins/elementor/includes/api.php:129
Stack trace:
#0 ../wp-content/plugins/elementor/includes/api.php(85): Elementor\Api::get_site_key()
#1 ../wp-content/plugins/elementor/includes/api.php(178): Elementor\Api::get_info_data()
#2 ../wp-content/plugins/elementor/modules/promotions/module.php(53): Elementor\Api::get_promotion_widgets()
#3 ../wp-includes/class-wp-hook.php(324): Elementor\Modules\Promotions\Module-&gt;Elementor\Modules\Promotions\{closure}(Object(Elementor\Widgets_Manager))
#4 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(NULL, Array)
#5 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#6 ../wp-content/plugins/elementor/includes/managers/widgets.php(146): do_action(&#039;elementor/widge...&#039;, Object(Elementor\Widgets_Manager))
#7 ../wp-content/plugins/elementor/includes/managers/widgets.php(345): Elementor\Widgets_Manager-&gt;init_widgets()
#8 ../wp-content/plugins/elementor/includes/elements/container.php(330): Elementor\Widgets_Manager-&gt;get_widget_types(&#039;image&#039;)
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(1502): Elementor\Includes\Elements\Container-&gt;_get_default_child_type(Array)
#10 ../wp-content/plugins/elementor/includes/base/element-base.php(369): Elementor\Element_Base-&gt;get_child_type(Array)
#11 ../wp-content/plugins/elementor/includes/base/element-base.php(1547): Elementor\Element_Base-&gt;add_child(Array)
#12 ../wp-content/plugins/elementor/includes/base/element-base.php(314): Elementor\Element_Base-&gt;init_children()
#13 ../wp-content/plugins/elementor/core/files/css/post.php(225): Elementor\Element_Base-&gt;get_children()
#14 ../wp-content/plugins/elementor/core/files/css/post.php(318): Elementor\Core\Files\CSS\Post-&gt;add_controls_stack_style_rules(Object(Elementor\Includes\Elements\Container), Array, Array, Array, Array)
#15 ../wp-content/plugins/elementor/core/files/css/post.php(297): Elementor\Core\Files\CSS\Post-&gt;render_element_styles(Object(Elementor\Includes\Elements\Container))
#16 ../wp-content/plugins/elementor/core/files/css/post.php(178): Elementor\Core\Files\CSS\Post-&gt;render_styles(Object(Elementor\Includes\Elements\Container))
#17 ../wp-content/plugins/elementor/core/files/css/base.php(687): Elementor\Core\Files\CSS\Post-&gt;render_css()
#18 ../wp-content/plugins/elementor/core/files/base.php(180): Elementor\Core\Files\CSS\Base-&gt;parse_content()
#19 ../wp-content/plugins/elementor/core/files/css/base.php(133): Elementor\Core\Files\Base-&gt;update_file()
#20 ../wp-content/plugins/elementor/core/files/css/base.php(229): Elementor\Core\Files\CSS\Base-&gt;update()
#21 ../wp-content/plugins/elementor/core/files/css/post.php(201): Elementor\Core\Files\CSS\Base-&gt;enqueue()
#22 ../wp-content/plugins/elementor/includes/frontend.php(1160): Elementor\Core\Files\CSS\Post-&gt;enqueue()
#23 ../wp-content/plugins/elementor/includes/frontend.php(1079): Elementor\Frontend-&gt;get_builder_content(40460)
#24 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend-&gt;apply_builder_in_content(&#039;&lt;img width=&quot;650...&#039;)
#25 ../wp-includes/plugin.php(205): WP_Hook-&gt;apply_filters(&#039;&lt;img width=&quot;650...&#039;, Array)
#26 ../wp-includes/feed.php(196): apply_filters(&#039;the_content&#039;, &#039;&lt;img width=&quot;650...&#039;)
#27 ../wp-includes/feed-rss2.php(104): get_the_content_feed(&#039;rss2&#039;)
#28 ../wp-includes/template.php(810): require_once(&#039;...&#039;)
#29 ../wp-includes/functions.php(1672): load_template(&#039;...&#039;)
#30 ../wp-includes/class-wp-hook.php(326): do_feed_rss2(false)
#31 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(NULL, Array)
#32 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#33 ../wp-includes/functions.php(1634): do_action(&#039;do_feed_rss2&#039;, false, &#039;rss2&#039;)
#34 ../wp-includes/template-loader.php(48): do_feed()
#35 ../wp-blog-header.php(19): require_once(&#039;...&#039;)
#36 ../index.php(17): require(&#039;...&#039;)
#37 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-11 05:31:20 [error X 2][../wp-content/plugins/elementor/includes/api.php::129] Uncaught Error: Call to a member function get_component() on null in ../wp-content/plugins/elementor/includes/api.php:129
Stack trace:
#0 ../wp-content/plugins/elementor/includes/api.php(85): Elementor\Api::get_site_key()
#1 ../wp-content/plugins/elementor/includes/api.php(178): Elementor\Api::get_info_data()
#2 ../wp-content/plugins/elementor/modules/promotions/module.php(53): Elementor\Api::get_promotion_widgets()
#3 ../wp-includes/class-wp-hook.php(324): Elementor\Modules\Promotions\Module-&gt;Elementor\Modules\Promotions\{closure}(Object(Elementor\Widgets_Manager))
#4 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(NULL, Array)
#5 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#6 ../wp-content/plugins/elementor/includes/managers/widgets.php(146): do_action(&#039;elementor/widge...&#039;, Object(Elementor\Widgets_Manager))
#7 ../wp-content/plugins/elementor/includes/managers/widgets.php(345): Elementor\Widgets_Manager-&gt;init_widgets()
#8 ../wp-content/plugins/elementor/includes/elements/container.php(330): Elementor\Widgets_Manager-&gt;get_widget_types(&#039;image&#039;)
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(1502): Elementor\Includes\Elements\Container-&gt;_get_default_child_type(Array)
#10 ../wp-content/plugins/elementor/includes/base/element-base.php(369): Elementor\Element_Base-&gt;get_child_type(Array)
#11 ../wp-content/plugins/elementor/includes/base/element-base.php(1547): Elementor\Element_Base-&gt;add_child(Array)
#12 ../wp-content/plugins/elementor/includes/base/element-base.php(314): Elementor\Element_Base-&gt;init_children()
#13 ../wp-content/plugins/elementor/core/files/css/post.php(225): Elementor\Element_Base-&gt;get_children()
#14 ../wp-content/plugins/elementor/core/files/css/post.php(318): Elementor\Core\Files\CSS\Post-&gt;add_controls_stack_style_rules(Object(Elementor\Includes\Elements\Container), Array, Array, Array, Array)
#15 ../wp-content/plugins/elementor/core/files/css/post.php(297): Elementor\Core\Files\CSS\Post-&gt;render_element_styles(Object(Elementor\Includes\Elements\Container))
#16 ../wp-content/plugins/elementor/core/files/css/post.php(178): Elementor\Core\Files\CSS\Post-&gt;render_styles(Object(Elementor\Includes\Elements\Container))
#17 ../wp-content/plugins/elementor/core/files/css/base.php(687): Elementor\Core\Files\CSS\Post-&gt;render_css()
#18 ../wp-content/plugins/elementor/core/files/base.php(180): Elementor\Core\Files\CSS\Base-&gt;parse_content()
#19 ../wp-content/plugins/elementor/core/files/css/base.php(133): Elementor\Core\Files\Base-&gt;update_file()
#20 ../wp-content/plugins/elementor/core/files/css/base.php(229): Elementor\Core\Files\CSS\Base-&gt;update()
#21 ../wp-content/plugins/elementor/core/files/css/post.php(201): Elementor\Core\Files\CSS\Base-&gt;enqueue()
#22 ../wp-content/plugins/elementor/includes/frontend.php(695): Elementor\Core\Files\CSS\Post-&gt;enqueue()
#23 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend-&gt;enqueue_styles(&#039;&#039;)
#24 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(NULL, Array)
#25 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#26 ../wp-includes/script-loader.php(2272): do_action(&#039;wp_enqueue_scri...&#039;)
#27 ../wp-includes/class-wp-hook.php(324): wp_enqueue_scripts(&#039;&#039;)
#28 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(&#039;&#039;, Array)
#29 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#30 ../wp-includes/general-template.php(3064): do_action(&#039;wp_head&#039;)
#31 ../wp-content/themes/astra/header.php(31): wp_head()
#32 ../wp-includes/template.php(810): require_once(&#039;...&#039;)
#33 ../wp-includes/template.php(745): load_template(&#039;...&#039;, true, Array)
#34 ../wp-includes/general-template.php(48): locate_template(Array, true, true, Array)
#35 ../wp-content/themes/astra/single.php(15): get_header()
#36 ../wp-includes/template-loader.php(106): include(&#039;...&#039;)
#37 ../wp-blog-header.php(19): require_once(&#039;...&#039;)
#38 ../index.php(17): require(&#039;...&#039;)
#39 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-12-14 06:51:24 [error X 1][../wp-content/plugins/elementor/includes/api.php::129] Uncaught Error: Call to a member function get_component() on null in ../wp-content/plugins/elementor/includes/api.php:129
Stack trace:
#0 ../wp-content/plugins/elementor/includes/api.php(85): Elementor\Api::get_site_key()
#1 ../wp-content/plugins/elementor/includes/api.php(178): Elementor\Api::get_info_data()
#2 ../wp-content/plugins/elementor/modules/promotions/module.php(53): Elementor\Api::get_promotion_widgets()
#3 ../wp-includes/class-wp-hook.php(324): Elementor\Modules\Promotions\Module-&gt;Elementor\Modules\Promotions\{closure}(Object(Elementor\Widgets_Manager))
#4 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(NULL, Array)
#5 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#6 ../wp-content/plugins/elementor/includes/managers/widgets.php(146): do_action(&#039;elementor/widge...&#039;, Object(Elementor\Widgets_Manager))
#7 ../wp-content/plugins/elementor/includes/managers/widgets.php(345): Elementor\Widgets_Manager-&gt;init_widgets()
#8 ../wp-content/plugins/elementor/includes/managers/elements.php(73): Elementor\Widgets_Manager-&gt;get_widget_types(&#039;uael-faq&#039;)
#9 ../wp-content/plugins/ultimate-elementor/modules/faq/module.php(98): Elementor\Elements_Manager-&gt;create_element_instance(Array)
#10 ../wp-includes/class-wp-hook.php(324): UltimateElementor\Modules\FAQ\Module-&gt;render_faq_schema(&#039;&#039;)
#11 ../wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters(NULL, Array)
#12 ../wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#13 ../wp-includes/general-template.php(3080): do_action(&#039;wp_footer&#039;)
#14 ../wp-content/themes/astra/footer.php(33): wp_footer()
#15 ../wp-includes/template.php(810): require_once(&#039;...&#039;)
#16 ../wp-includes/template.php(745): load_template(&#039;...&#039;, true, Array)
#17 ../wp-includes/general-template.php(92): locate_template(Array, true, true, Array)
#18 ../wp-content/themes/astra/page.php(44): get_footer()
#19 ../wp-includes/template-loader.php(106): include(&#039;...&#039;)
#20 ../wp-blog-header.php(19): require_once(&#039;...&#039;)
#21 ../index.php(17): require(&#039;...&#039;)
#22 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Ultimate Addons for Elementor Pro: Compatibilité non spécifiée

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @Flow-CH!

Thank you for submitting your issue! 🙏 

Please keep in mind that the **Elementor Beta (Developer Edition)** plugin is meant to be used **<ins>only on development or staging environments</ins>** by Addon, Theme, and Plugin developers to test their software against the latest version of Elementor, and is **<ins>not supposed to be installed on Production websites</ins>** as it is not stable for production. Never install this plugin on a live website.

Follow the instructions in the article below to learn how to revert to a stable version of Elementor:
https://elementor.com/help/revert-to-release-from-developer-edition/

We are going to investigate your report, and if needed we will patch our code. For now, follow the instructions in the article above to solve the issue on your installation.

Kind regards

**Comment by Flow-CH** [comment]
Ok. Thank you and best regards, Laurence

**Comment by nicholaszein** [comment]
Hi @Flow-CH!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved with the release of Elementor v3.26.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
